### PR TITLE
Added volunteer type and count value

### PR DESCRIPTION
**Pull request by ChaudharyShiva18**

### CODE REVIEW

File: src/services/columns.js [hunk 1]
```diff
@@ -150,6 +152,39 @@ export async function getVolunteerCard(req) {
...
+  )
+    .then((response) => Promise.resolve(response))
+    .catch(function (err) {
+      return Promise.reject(err);
+    });
+}
+
+export async function getCount() {
+  // let url = `https://www.covidindiahelp.info:3001/tags`;
+  let url = `https://www.covidindiahelp.info:3002/count`;
```
Comment (ayushkul) [inline src/services/columns.js]: Connect with API gateway

**Review by ayushkul — CHANGES_REQUESTED** [review]

File: src/services/columns.js (hunk 1 above)
Comment (ChaudharyShiva18) [reply src/services/columns.js]: Done

File: src/modules/leadDetails/index.js [hunk 2]
```diff
@@ -1,132 +1,210 @@
...
-        if (!data && !data.phoneNumber.length) {
-            return []
-        }
-        let uniqueContact = []
-        data.phoneNumber.map((item) => {
-            item = item.trim()
-            if (!uniqueContact.includes(item) && item.length >= 10) uniqueContact.push(item)
-        })
-        return uniqueContact;
+  };
```
Comment (lhtdeepaktomar) [inline src/modules/leadDetails/index.js:56]: why removed this function?

File: src/modules/leadDetails/index.js [hunk 3]
```diff
@@ -1,132 +1,210 @@
...
-        try {
-            let response = await upVote(data, desription)
-            if (!response || !response.responseData || Object.keys(response.responseData).length < 1)
-                return;
-            this.setState({leadDetails: response.responseData, workingComment:''})
-
-        } catch (error) {
-            console.log(error)
-        }
-        this.handleWorkingPopoverClose();
```
Comment (lhtdeepaktomar) [inline src/modules/leadDetails/index.js:76]: why removed this function also

**Pull request merged**